src/websocket.js

```diff
@@ -6,29 +6,35 @@ import openWebSocket from 'open-websocket'
 const BASE = 'wss://stream.binance.com:9443/ws'
 const FUTURES = 'wss://fstream.binance.com/ws'
 
-const depth = (payload, cb) => {
+const depthTransform = m => ({
+  eventType: m.e,
+  eventTime: m.E,
+  symbol: m.s,
+  firstUpdateId: m.U,
+  finalUpdateId: m.u,
+  bidDepth: m.b.map(b => zip(['price', 'quantity'], b)),
+  askDepth: m.a.map(a => zip(['price', 'quantity'], a)),
+})
+
+const futuresDepthTransform = m => ({
+  eventType: m.e,
+  eventTime: m.E,
+  transactionTime: m.T,
+  symbol: m.s,
+  firstUpdateId: m.U,
+  finalUpdateId: m.u,
+  prevFinalUpdateId: m.pu,
+  bidDepth: m.b.map(b => zip(['price', 'quantity'], b)),
+  askDepth: m.a.map(a => zip(['price', 'quantity'], a)),
+})
+
+const depth = (payload, cb, transform = true, variator) => {
```

Comment:
Since the `variator` is always provided, the `transform` param should be the last parameter in order to make use of the default value feature. This will be valid for all the other socket methods

Reply:
Or maybe it's better to add an empty line to the default variator value? Because it is unlikely that someone will use it outside the library, but at least I use the transformation parameter, and if you change places, you will always need to first set the variator parameter if we want to use the transformation parameter with the value FALSE.

Reply:
Now we call like this:

```
const clean = client.ws.depth('ETHBTC', depth => {
  console.log(depth)
}, false)
```

And if we change the order, it will be necessary to call like this:

```
const clean = client.ws.depth('ETHBTC', depth => {
  console.log(depth)
}, '', false)
```

Reply:
Then maybe make it so the 3rd parameter is an object with a transform and variator props. Only giving false is a little too obscure probably?

Reply:
Yes, that would be clearer. But advise how then to apply it to the "user" method? 
const user = (opts, variator) => (cb, transform) => {
https://github.com/anwebdevelopers/binance-api-node/blob/2f7aa34b0b0eb348ae26ece4387a8f4f63322101/src/websocket.js#L463

Reply:
Heh I guess maybe it's fine, since you're not supposed to manually change the variator anyway

Reply:
Excellent, thank you!)